buddyboss/buddyboss-platform
# PROD-742 Activity link preview issue during privacy update - Fixed

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2914  .

### How to test the changes in this Pull Request:

1. Go to Activity Feed
2. Paste any link (make sure that the link previews is enabled)
3. Once posted, change the privacy to All members or any privacy
4. Reload the page
5. See the error is not happening

### Proof Screenshots or Video

https://www.loom.com/share/73097cb6c4da4ddeaf1946f8f77aa079

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
